components/infra-proxy-service/server/clients.go

```diff
@@ -15,12 +15,12 @@ import (
 // GetClients get clients list
 func (s *Server) GetClients(ctx context.Context, req *request.Clients) (*response.Clients, error) {
 
-	client, err := s.createClient(ctx, req.OrgId)
```

Comment:
Now `server.createClient` returning wrap `ChefClient`
In order to access the `chef.Client` required extra `client` object navigation.